plugins/services/src/js/structs/__tests__/Service-test.js

```diff
@@ -32,6 +32,32 @@ describe("Service", function() {
     });
   });
 
+  describe("#getTaskCount", function() {
+    it("returns the number of reported tasks", function() {
+      const service = new Service({
+        tasks: [{ foo: "bar" }, { bar: "baz" }]
+      });
+
+      expect(service.getTaskCount()).toEqual(2);
+    });
+
+    it("returns the number of reported tasks", function() {
```

Comment:
Let's rename this test to something different from the one above. It'll be easier to read the test from the test reporter to understand what it was testing and to find the exact test in code (being unique).

Perhaps `returns 0 when task array is empty`. We already have context for the method given the parent `describe` so we should name this something relevant to the test case.